aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
+	"id": "AWS061",
+	"apiVersion": 2,
+	"version": 1,
+	"title": "API Gateway stages for V1 and V2 should have access logging enabled",
+	"description": "Ensures that Amazon API Gateway API stages have Amazon CloudWatch Logs enabled.",
+	"custom": {
+		"severity": "MEDIUM",
+		"possible_Impact": "Logging provides vital information about access and usage.",
+		"urls": ["https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html", "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apigatewayv2_stage#access_log_settings"],
```

Comment:
We're planning on removing Terraform-specific links and adding these within tfsec, so we can have cloud formation ones specifically added in cfsec etc. Personally I think the central links should be generic and relate to the security issue in general, not the IaC that was used to define it.